scripts/build/build.ps1

```diff
@@ -33,8 +33,8 @@ else {
 header "Compiling"
 _ "$RepoRoot\scripts\compile\compile.ps1" @("$Configuration")
 
-header "Running Tests"
-_ "$RepoRoot\scripts\test\runtests.ps1"
+header "Running Tests on Stage 2"
```

Comment:
Can we set the path right here? This way the runtests script would be capable of testing any set of binaries.